HW_five/Task_5_1.py

```diff
@@ -0,0 +1,11 @@
+# Напишите программу, которая на вход принимает два числа A и B,
+# и возводит число А в целую степень B с помощью рекурсии.
+
+def degree(a, b): #задаем рекурсию
+    if b == 0:
+        return 1 # выход из цикла рекурсии
+    return(a * degree(a, b-1))
```

Comment:
Скобки не нужны. `return` это не функция.
```suggestion
    return a * degree(a, b-1)
```